motr/m0crate/tests/test1_io.yaml

```diff
@@ -37,6 +37,7 @@ LOG_LEVEL: 4  # err(0), warn(1), info(2), trace(3), debug(4)
 WORKLOAD_SPEC:               # Workload specification section
```

Comment:
Please verify this yaml file if it works.
Please see the line #35:
```
LOG_LEVEL: 4 # err(0), warn(1), info(2), trace(3), debug(4)
```
It should be in the section of "MOTR_CONFIG". But now it is on the top level.
I think this will introduce error.
Please check this.

Reply:
I have tested this and it works, would you like me to change it?

Reply:
Yes. Please change the alignment issue.

Reply:
I made a commit to fix the alignment issue.